src/icons/fileIcons.ts

```diff
@@ -882,5 +882,7 @@ export const fileIcons: FileIcons = {
         { name: 'sketch', fileExtensions: ['sketch'] },
         { name: 'adonis', fileNames: ['.adonisrc.json'] },
         { name: 'meson', fileNames: ['meson.build'] },
+        { name: 'tsconfig', fileNames: ['tsconfig.json'] },
+        { name: 'jsconfig', fileNames: ['jsconfig.json'] }
```

Comment:
This also needs `fileExtensions`, since `foo.tsconfig.json` is a semi‑common way to differentiate multiple **TypeScript** configurations.

The other is `tsconfig.foo.json`.

```suggestion
        { name: 'tsconfig', fileNames: ['tsconfig.json'], fileExtensions: ['tsconfig.json'] },
        { name: 'jsconfig', fileNames: ['jsconfig.json'], fileExtensions: ['jsconfig.json'] }
```